### DISCUSSION TITLE
Laravel login page redirect loop

### DISCUSSION THREAD

**Discussion by lipiniak** [beyondcode/herd-community#226] [Q&A]
I have very old project in laravel 6, when i added Herd to my MacOs application wont log in. When i put credentials and click log in i have 302 on my dashboard page and getting back to login. 
Should i have add something to nginx config file? It looks like nginx not allowing to redirect, and returning to login page.

**Comment by Lirux** [comment]
What PHP version is set for your Laravel 6 project? Possible that it is set to PHP 8.x. and Laravel 6 is not working with PHP8?!

**Comment by zaimazhar** [comment]
Bumping since I am having the same issue to on Mac. I was instantly redirected back to my login page after the successful login.

```
"php": "^8.1",
"laravel/framework": "^10.0",
```

<img width="744" alt="image" src="https://github.com/user-attachments/assets/00fd67c8-0901-4ec3-a6f9-a79686634b8b">

**Comment by lipiniak** [comment]
Hi, @zaimazhar i didn't resolved this problem. I switched back to Laravel Valet. 
For now i think that was problem with PHP versions.

**Comment by zaimazhar** [comment]
@lipiniak Hey, just to let you know I resolved the problem on my side. Mine was simply changing the `APP_ENV` from `staging` to `local` then I can login into the dashboard of my application.
